server/src/seeds/testdata.ts

```diff
@@ -225,7 +225,7 @@ const createTestTasks = async () => {
       ratings: defaultRatings.map(({ createdBy, createdFor, dimension }) =>
         randomTaskratingValue(createdBy, createdFor, dimension),
       ),
-      completed: true,
+      status: 'COMPLETED',
```

Comment:
```suggestion
      status: TaskStatus.COMPLETED,
```

We should always use the variants through the enum, not directly as the bare values.
This also removes the need for those any casts below.